wiredep-cli.js

```diff
@@ -54,7 +54,7 @@ if (!argv.s && !argv.src) {
 
 if (argv.b || argv.bowerJson) {
   try {
-    argv.bowerJson = JSON.parse(fs.readFileSync(argv.b || argv.bowerJson));
+    argv.b = JSON.parse(fs.readFileSync(argv.b || argv.bowerJson));
```

Comment:
I think all we need here is:

```diff
if (argv.b || argv.bowerJson) {
  try {
      argv.bowerJson = JSON.parse(fs.readFileSync(argv.b || argv.bowerJson));
+     delete argv.b;
  } catch (e) {}
}
```

The problem is later in the script with this code:

```js
if (argObj.short === arg) {
  acc[argObj.full] = argv[arg];
  delete acc[arg];
}
```

This block is responsible for taking all of the shorthand input and renaming it to what the wiredep API expects.

Earlier, when `.b` was transformed into `.bowerJson`, we didn't remove `.b`. That tricked this code into thinking there is only a `.b` and it needs to be renamed to its full name, `.bowerJson`. So, basically, it just overwrites what we did earlier.

Sorry for the iffy code :(